From a5cc30c425040cb352aa185993afac0c846f18d4 Mon Sep 17 00:00:00 2001
From: Cyril Hrubis <chrubis@suse.cz>
Date: Wed, 6 Mar 2013 16:39:03 +0100
Subject: [PATCH] syscalls/sysctl03: Change TWARN to TINFO.

The TWARN is propagated to the test return value
and interpreted as a test (internal) failure,
moreover the rest of the testcases seems to
use TWARN this way.

So this replaces TWARN with TINFO which
makes the testcase to PASS.

Signed-off-by: Cyril Hrubis <chrubis@suse.cz>
---
 testcases/kernel/syscalls/sysctl/sysctl03.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/testcases/kernel/syscalls/sysctl/sysctl03.c b/testcases/kernel/syscalls/sysctl/sysctl03.c
index 6caaa72..6a2cae8 100644
--- a/testcases/kernel/syscalls/sysctl/sysctl03.c
+++ b/testcases/kernel/syscalls/sysctl/sysctl03.c
@@ -117,7 +117,7 @@ int main(int ac, char **av)
 		exp_eno = EPERM;
 	} else {
 		/* ^^ Look above this warning. ^^ */
-		tst_resm(TWARN,
+		tst_resm(TINFO,
 			 "this test's results are based on potentially undocumented behavior in the kernel. read the NOTE in the source file for more details");
 		exp_eno = EACCES;
 		exp_enos[0] = EACCES;
-- 
1.7.8.6

